client/src/components/pipeline/component/list.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
about code convention, how about make all js quote a `'`. eg. `import React from 'react'`, this is more common in js project

Reply:
I'd suggest this as our style conventions: https://github.com/airbnb/javascript